lib/tasks/set_auth_bypass_id.rake

```diff
@@ -1,6 +1,8 @@
 desc "Imports services and removes any old ones"
 task set_auth_bypass_id: :environment do
   Edition.all.each do |edition|
-    edition.update(auth_bypass_id: edition.temp_auth_bypass_id)
+    # rubocop:disable Rails/SkipsModelValidations
+    edition.update_attribute(:auth_bypass_id, edition.temp_auth_bypass_id)
+    # rubocop:enable Rails/SkipsModelValidations
```

Comment:
any reason to not use `update!` ?

Reply:
That'll raise an exception if the validation fails, whereas I want it to silently ignore the validation errors.

Reply:
Oh yeah 🤦  I didn't really consider that it could fail on other fields than auth_bypass_id